content/hi/docs/reference/glossary/wg.md

```diff
@@ -0,0 +1,18 @@
+---
+title: WG 
```

Comment:
Please check how other localisations have translated/transliterated it & I'd advise the same be affected here.

Reply:
WG is termed as WG in most of the other translations.